source/get-started/cloud-install/azure.rst

```diff
@@ -18,25 +41,12 @@ is offered with three different images, also known as a
   tools for developing machine learning applications.
 
 You can access these images directly from your MS Azure dashboard through the
-`Azure portal`_ or by using the MS Azure :abbr:`CLI (Command Line Interface)`
-2.0. If you do not already have an account set up with MS Azure, you can sign
-up for a `MS Azure free account`_ to access the |CL|
-:abbr:`VM(Virtual Machine)` images.
-
-The Azure CLI offers the ability to create and manage resources in MS Azure
-from the command line. In this tutorial you learn to:
-
-#. Install the latest MS Azure CLI on your |CL| machine.
-
-#. Log into MS Azure using the CLI 2.0 interface.
-
-#. Create a MS Azure resource group.
-
-#. Locate the |CL| images.
+`Azure portal`_ or by using the MS Azure CLI 2.0. If you do not already have an
+account set up with MS Azure, you can sign up for a `MS Azure free account`_
+to access the |CL| :abbr:`VM (Virtual Machine)` images.
```

Comment:
Sign in at the `Azure portal`_ to access these images from the Azure dashboard, or use the MS Azure CLI 2.0.

Reply:
Done.